### PR TITLE
Update arc.js 56 ; use polling

### PR DESCRIPTION
**Pull request by roienatan**
resolves: https://github.com/daostack/alchemy/issues/2126 - for 2.0
resolves: https://github.com/daostack/alchemy/issues/2049

- **Creating proposals**, **voting** and **redeeming** are reflected in the UI automatically
- **Staking** and **number of Regular Proposals, Pending Boosting and Boosted Proposals** are still not reflected in the UI (need to refresh the page to see)

**UPDATE:** proposals counters are now fixed.
**UPDATE:** staking data is now updated automatically

### CODE REVIEW

**Comment by roienatan** [comment]
@dkent600 The issue with the proposal count (https://github.com/daostack/alchemy/issues/2049) happens because `pluginState` in not updated. I'm not sure it's related to the polling.

**Comment by dkent600** [comment]
> @dkent600 The issue with the proposal count (#2049) happens because `pluginState` in not updated. I'm not sure it's related to the polling.

If Alchemy is supplying subscribe:true when it requests the proposalState, then the state should be updated by polling.

**Comment by roienatan** [comment]
> > @dkent600 The issue with the proposal count (#2049) happens because `pluginState` in not updated. I'm not sure it's related to the polling.
> 
> If Alchemy is supplying subscribe:true when it requests the proposalState, then the state should be updated by polling.

I say this because I tried to replace **all** `subscribe: true` to `polling: true` and still the counters didn't update. That's why I might think that there is a problem in Alchemy code itself (which for now I didn't find) and that's why I asked you if you recall that those counters were properly working before.

**Review by orenyodfat — CHANGES_REQUESTED** [review]
The following lines need to be updated : 
https://github.com/daostack/alchemy/blob/update-arcjs-56/src/components/Account/AccountProfilePage.tsx#L358

https://github.com/daostack/alchemy/blob/update-arcjs-56/src/components/Plugin/ContributionRewardExtRewarders/Competition/List.tsx#L140

**Comment by orenyodfat** [comment]
> > > @dkent600 The issue with the proposal count (#2049) happens because `pluginState` in not updated. I'm not sure it's related to the polling.
> > 
> > 
> > If Alchemy is supplying subscribe:true when it requests the proposalState, then the state should be updated by polling.
> 
> I say this because I tried to replace **all** `subscribe: true` to `polling: true` and still the counters didn't update. That's why I might think that there is a problem in Alchemy code itself (which for now I didn't find) and that's why I asked you if you recall that those counters were properly working before.

how about updating this as well ? https://github.com/daostack/alchemy/blob/update-arcjs-56/src/components/Dao/ProposalPluginCard.tsx#L103 ?

File: src/components/Plugin/ContributionRewardExtRewarders/Competition/utils.ts [hunk 1]
```diff
@@ -201,7 +201,7 @@ export const getSubmissionVoterHasVoted = (submissionId: string, voterAddress: s
 
 // export const primeCacheForSubmissionsAndVotes = (): Observable<any> => {
 //   return combineLatest(
-//     CompetitionSuggestion.search(getArc(), {}, { subscribe: true, fetchAllData: true }),
-//     CompetitionVote.search(getArc(), {}, { subscribe: true, fetchAllData: true })
+//     CompetitionSuggestion.search(getArc(), {}, { polling: true, fetchAllData: true }),
```
Comment (orenyodfat) [inline src/components/Plugin/ContributionRewardExtRewarders/Competition/utils.ts:204]: lets remove these comments lines ?

**Review by orenyodfat — CHANGES_REQUESTED** [review]
lets set the polling time to 30 seconds and have it on a single global var

**Pull request merged**